substrate/bin/node/runtime/src/constants.rs

```diff
@@ -63,7 +63,11 @@ pub mod time {
 
 	// NOTE: Currently it is not possible to change the epoch duration after the chain has started.
 	//       Attempting to do so will brick block production.
+	#[cfg(not(feature = "staking-playground"))]
 	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 10 * MINUTES;
+	#[cfg(feature = "staking-playground")]
+	pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 2 * MINUTES;
```

Comment:
Would be possible to increase to 4 minutes instead? I find especially the SignPhase a bit short for doing proper testing